documentation/Sphinx/user_guide/output/post-processing.rst

```diff
@@ -0,0 +1,100 @@
+.. -----------------------------------------------------------------------------
+     (c) Crown copyright 2024 Met Office. All rights reserved.
+     The file LICENCE, distributed with this code, contains details of the terms
+     under which the code may be used.
+   -----------------------------------------------------------------------------
+
+Post-processing
+---------------
+
+The file 'post-processing.py' can be used for processing the raw outputs 
+into a more usable format, merging all MPI ranks to give the means
+across all ranks.
+
+Please note that you will need an environment with the pandas library. This is 
+included within the Vernier conda environment, and future versions of this
+script will require additional libraries.
+
+.. dropdown:: ``Creating/ loading conda environments in Bash``
+	
+  1. Navigate to the .yml file
+
+  .. code-block:: shell
+
+    cd etc/conda/
+
+  2. Create the environment from the .yml file
+
+  .. code-block:: shell
+
+    conda env create -f vernier-env.yml
+
+  3. Activate the environment
+
+  .. code-block:: shell
+
+    conda activate vernier-env
+
```

Comment:
This can be simplified to
```suggestion
.. dropdown:: ``Creating/ loading conda environments in Bash``

  .. code-block:: shell

    conda env create -f ./etc/conda/vernier-env.yml
    conda activate vernier-env

```